pernstrong/whats-cookin
# Feature/search

#### What's this PR do?
It lets the user search both favorite recipes and all recipes and disregards capital letters in the search. It also formats the pantry display and cleans up some of the code.
#### Where should the reviewer start?
scripts.js, all other files for the clean up
#### How should this be manually tested?
The user can type into the search box to see if the search functions properly. Also, click on the display pantry button to see if formatted correctly.
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
